vertx-mongo-client/src/main/java/examples/Examples.java

```diff
@@ -547,4 +553,268 @@ public void example16_d1(MongoClient mongoClient) throws Exception{
 
     });
   }
+  public void example17(MongoClient mongoClient) {
+    mongoClient.createGridFsBucketService("fs", res -> {
+      if (res.succeeded()) {
+        MongoGridFsClient client = res.result();
+        //Interact with the GridFS client...
+      } else {
+        res.cause().printStackTrace();
+      }
```

Comment:
Printing Stacktraces.... OK I see this is following the same code style, happy to let this go.